File: PSI-2020-backend/PSI-2020-backend/Models/Database/LearningEffect.cs
using PSI_2020_backend.Models.Database.Bases;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PSI_2020_backend.Models.Database
{
    public class LearningEffect : ModelBase
    {
        public string Code { get; set; }
        public string Description { get; set; }
        public string DescriptionEng { get; set; }
        public EffectCategory Category { get; set; }
    }
}

File: frontend/src/app/models/learning-effect.ts
import { EffectCategory } from "./effect-category";

export class LearningEffect {
    id: string;
    code: string;
    description: string;
    descriptionEng: string;
    category: EffectCategory;

    constructor(init?: Partial<LearningEffect>) {
        Object.assign(this, init);
      }
}
